### ISSUE TITLE
something is wrong error after LOOT update today

### ISSUE THREAD

**Issue by gshillitani** [loot/loot#1072]
[LOOTDebugLog.txt](https://github.com/loot/loot/files/2775965/LOOTDebugLog.txt)

Something is wrong error with both Fallout 4 and with Fallout New Vegas. Have uninstalled/reinstalled LOOT, reinstalled F4SE, run the game via regular loader and via F4SE loader, reinstalled Vortex, started LOOT from both Vortex and from .exe, made certain plugins.txt is not read-only. When I first open LOOT, I get this
![image](https://user-images.githubusercontent.com/8304720/51432394-1db00580-1c05-11e9-9e69-82e0829d1c2a.png)
then this
![image](https://user-images.githubusercontent.com/8304720/51432403-433d0f00-1c05-11e9-939e-9f37c8a5cbc1.png)

**Comment by CptMcSplody** [comment]
```
[16:06:47.718075] [trace]: Evaluating condition: version("LOOT", "0.13.1.0", <) and file("cc[A-Z]{3}FO4[0-9]{3}-[a-zA-Z0-9()]+\.es(l|m)")
[16:06:47.719002] [trace]: Evaluating condition: file("../Fallout4.exe") and version("../Fallout4.exe", "1.10.111.0", <)
[16:06:47.719817] [trace]: Evaluating condition: file("../f4se_loader.exe") and version("../f4se_loader.exe", "0.0.6.12", <)
[16:06:47.720512] [trace]: Evaluating condition: not version("../f4se_loader.exe", "0.0.6.12", <)
[16:06:47.720553] [trace]: Evaluating condition: version("../f4se_loader.exe", "0.0.6.12", ==) and version("../Fallout4.exe", "1.10.111.0", !=)
[16:06:47.721159] [trace]: Evaluating condition: file("F4SE/Plugins/place.dll") and ((version("F4SE/Plugins/place.dll", "1.18.0.1114", ==) and version("../Fallout4.exe", "1.10.114.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1114", <) and version("../Fallout4.exe", "1.10.114.0", ==)))
[16:06:47.721579] [error]: Failed to evaluate condition "file("F4SE/Plugins/place.dll") and ((version("F4SE/Plugins/place.dll", "1.18.0.1114", ==) and version("../Fallout4.exe", "1.10.114.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1114", <) and version("../Fallout4.exe", "1.10.114.0", ==)))". Details: An error was encountered while reading the version fields of "C:\Program Files (x86)\Steam\steamapps\common\Fallout 4\Data\F4SE/Plugins/place.dll": entry not found
[16:06:47.721671] [error]: Exception while executing query: Failed to evaluate condition "file("F4SE/Plugins/place.dll") and ((version("F4SE/Plugins/place.dll", "1.18.0.1114", ==) and version("../Fallout4.exe", "1.10.114.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1114", <) and version("../Fallout4.exe", "1.10.114.0", ==)))". Details: An error was encountered while reading the version fields of "C:\Program Files (x86)\Steam\steamapps\common\Fallout 4\Data\F4SE/Plugins/place.dll": entry not found
```

**Comment by gshillitani** [comment]
Which mod? F4SE?
[f4se_0_06_15.zip](https://github.com/loot/loot/files/2775996/f4se_0_06_15.zip)

**Comment by Madpaddy** [comment]
https://www.nexusmods.com/fallout4/mods/9424

**Comment by CptMcSplody** [comment]
@gshillitani @Madpaddy I pushed an update to resolve this temporarily, you'll need to delete your local copy of the masterlist. `Users\%username%\AppData\Local\LOOT\Fallout4`.

**Comment by Madpaddy** [comment]
Ok cheers, it wont delete my load order will it really dont want to have to set the orders up again.

**Comment by CptMcSplody** [comment]
No it won't change your load order. Just delete the `masterlist.yaml`, you can backup your `userlist.yaml` if you have any custom rules.

**Comment by Madpaddy** [comment]
Fantastic cheers for the fast fix...

**Comment by gshillitani** [comment]
I saw that in the log and verified the place.dll was in the correct place 
in the directory... weird. I'm out right now but when I get home I'll 
download the patch. Thanks, you guys have been very helpful 😀

**Comment by Ortham** [comment]
OK, so this is happening because the DLL parser assumes that the version info metadata has been added with language code 1033 (US English) so it's looking for that, but `place.dll` has its language code set to 1049 (Russian), though its manifest uses 1033. That's why the "not found" error is happening - it's the metadata in the file that's not found, not the file itself (that would have a different error).

**Comment by Madpaddy** [comment]
> 
> 
> OK, so this is happening because the DLL parser assumes that the version info metadata has been added with language code 1033 (US English) so it's looking for that, but `place.dll` has its language code set to 1049 (Russian), though its manifest uses 1033. That's why the "not found" error is happening - it's the metadata in the file that's not found, not the file itself (that would have a different error).

Weirdly it only seemed to have a problem with it after the update though.

**Comment by Ortham** [comment]
Right, I've got a fix for this now, I'll link to the updated libloot DLL in the morning once it's built so the fix can be verified.

**Comment by gshillitani** [comment]
Works perfectly after the patch :D Thanks so much!


> @gshillitani @Madpaddy I pushed an update to resolve this temporarily, you'll need to delete your local copy of the masterlist. `Users\%username%\AppData\Local\LOOT\Fallout4`.

**Comment by Ortham** [comment]
OK, this `loot.dll` should fix the issue: https://bintray.com/loot/snapshots/download_file?file_path=libloot-0.14.1-4-ga7bf878_update-dependencies-win32.7z

**Comment by gshillitani** [comment]
I get a 404 when I click on the link directly, and if I right click to 'save as' it wants to save as the html page, not the 7z file?

**Comment by gshillitani** [comment]
> 
> 
Thanks! and thanks to everyone for solving the issue so quickly!!


> Sorry, the link changed, use this one: https://bintray.com/loot/snapshots/download_file?file_path=libloot-0.14.2-0-g4f01487_dev-win32.7z

**Comment by Ortham** [comment]
Fixed in 119aa27924632a19449126fd9aaeca4f9fc54886.
